zds/utils/templatetags/remove_url_scheme.py

```diff
@@ -5,8 +5,8 @@
 register = template.Library()
 
 
-@register.filter('remove_url_protocole')
-def remove_url_protocole(input_url):
+@register.filter('remove_url_scheme')
+def remove_url_scheme(input_url):
```

Comment:
Tant que tu y es, ça vaudrait la peine de remplacer les `in` par des `startsWith` ici https://github.com/artragis/zds-site/blob/b3a18d9f12d613b17af00520eb89fd21e433df60/zds/utils/templatetags/remove_url_scheme.py#L13 

Parce que là c'est buggé actuellement, d'une façon assez évidente. `http://example.com?q=http://zestedesavoir.com` donne

<img width="451" alt="screen shot 2017-08-12 at 18 13 19" src="https://user-images.githubusercontent.com/2022803/29242269-0bf9e5d6-7f8a-11e7-9e71-fe00a0bcb390.png">